kubernetes/skaffold/src/main/kotlin/com/google/kubernetes/tools/skaffold/editing/SkaffoldSchemaProviderFactory.kt

```diff
@@ -27,6 +27,7 @@ import com.jetbrains.jsonSchema.extension.JsonSchemaProviderFactory
 class SkaffoldSchemaProviderFactory : JsonSchemaProviderFactory {
     override fun getProviders(project: Project): MutableList<JsonSchemaFileProvider> {
         return mutableListOf(
+                SkaffoldSchemaFileProvider("v1beta7"),
```

Comment:
Looks like this has not been released yet? Should we wait until official release of Skaffold with this schema?

Reply:
oh I didn't catch that. Yeah. lets hold off on merging.